portfolio/src/main/webapp/script.js

```diff
@@ -13,16 +13,69 @@
 // limitations under the License.
 
 /**
- * Adds a random greeting to the page.
+ * Gets comments for the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function getComments() {
+  const commElem = document.getElementById('comments-container');
+  commElem.innerHTML = "";
+  var number = document.getElementById("number").value;
+  fetch("/data?number=" + number).then(response => response.json()).then((comments) => {
+    const commElem = document.getElementById('comments-container');
+     comments.forEach(c => {
```

Comment:
Good use of one-character variable names!

Reply:
Thanks! Didn't know this was a good thing, I just did it out of convenience. Are one character names best practice when using .forEach or .map functions in js?

Reply:
Yeah, quoting from our style guide: "The brevity of single letter variables is especially appealing in lambda expressions."

Other examples of common single-letter variables: 
- i, j, k for counters in loops
- e for exceptions in catch blocks
- r, g, b when referring to an RGB color space

Reply:
Cool thanks!